### PR TITLE
MES-1963  - finish driving test

### PR DESCRIPTION
**Pull request by roberthall12**
## Description and relevant Jira numbers

Creating the Back to Office page as per the abstract design,
https://jira.i-env.net/browse/MES-1963

## Pull Request checklist:

- [x] [WIP] tag removed from PR title
- [x] PR has an understandable description

## Git feature branch checklist:

- [x] branch name comply with our branching strategy
- [x] git branch contains relevant JIRA ticket number
- [x] branch rebased against the latest develop
- [x] commits are squashed

## Sign off process checklist:

- [x] Code has been tested manually
- [x] Tests are passing
- [x] PR link added to JIRA
- [x] Reviewers selected in Github
- [ ] Any new reusable component/widget added to component library on Confluence

- [ ] Screenshot(s) captured on the physical device (if applicable)
- [ ] Tested by QA
- [ ] PO's approval

Screen Shot 2019-04-03 at 10 05 38

### CODE REVIEW

File: src/pages/back-to-office/__tests__/back-to-office.spec.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    // Unit tests for the components TypeScript class
+    it('should create', () => {
+      expect(component).toBeDefined();
+    });
+  });
+
+  describe('DOM', () => {
+    describe('popToRoot', () => {
+      it('should call the popToRoot method in the navcontroller', () => {
+        component.popToRoot();
```
Comment (johnmorleybjss) [inline src/pages/back-to-office/__tests__/back-to-office.spec.ts]: This is a component class test, not a DOM test.

File: src/pages/back-to-office/back-to-office.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
+page-back-to-office {
+  div {
+    background-color: #006f4b;
```
Comment (johnmorleybjss) [inline src/pages/back-to-office/back-to-office.scss]: Might be worth having this as a colour variable.

File: src/pages/back-to-office/back-to-office.scss [hunk 3]
```diff
@@ -0,0 +1,26 @@
+page-back-to-office {
```
Comment (johnmorleybjss) [inline src/pages/back-to-office/back-to-office.scss]: Feels like this the CSS for this page could be made a bit more concise with flexbox.

File: src/pages/back-to-office/back-to-office.analytics.effects.ts [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+} from './back-to-office.actions';
+
+@Injectable()
+export class BackToOfficeAnalyticsEffects {
+  constructor(
+    public analytics: AnalyticsProvider,
+    private actions$: Actions,
+  ) {
+    this.analytics.initialiseAnalytics()
+          .then(() => {})
```
Comment (szabolcs-keresztes) [inline src/pages/back-to-office/back-to-office.analytics.effects.ts]: We don't need to handle the happy path if there isn't anything to do

File: src/pages/office/__tests__/office.spec.ts [hunk 5]
```diff
@@ -41,6 +43,11 @@ describe('OfficePage', () => {
   });
 
   describe('DOM', () => {
-    // Unit tests for the components template
+    describe('popToRoot', () => {
+      it('should call the popToRoot method in the navcontroller', () => {
+        component.popToRoot();
```
Comment (szabolcs-keresztes) [inline src/pages/office/__tests__/office.spec.ts]: This is a class test rather than a DOM

File: src/pages/office/office.html [hunk 6]
```diff
@@ -1,22 +1,22 @@
...
-      </ion-card-title>
-    </ion-card-header>
-    <ion-card-content>
-      <button ion-button (click)="popToRoot()">Journal Page</button>
-      <button ion-button navPush="TerminateTestPage">Terminate Test</button>
-    </ion-card-content>
-  </ion-card>
-</ion-content>
+    <ion-navbar>
+      <ion-title>Office</ion-title>
```
Comment (szabolcs-keresztes) [inline src/pages/office/office.html]: No need for these indentations.

**Pull request merged**